src/dsp/pdr.c

```diff
@@ -1628,7 +1628,9 @@ int pldm_entity_association_pdr_create_new(pldm_pdr *repo,
 	pldm_msgbuf_extract(src_c, container_id);
 	pldm_msgbuf_insert(dst, container_id);
 	container_id = htole16(container_id);
-	memcpy(container_id_addr, &container_id, sizeof(uint16_t));
+	if (container_id_addr != NULL) {
```

Comment:
why is this change made?

Reply:
some new gcc changes is causing clang tidy to fail so I made this change in commit upstream and here too